pkg/controller/garbagecollector/garbagecollector_test.go

```diff
@@ -213,12 +223,27 @@ func setupGC(t *testing.T, config *restclient.Config) garbageCollector {
 	if err != nil {
 		t.Fatal(err)
 	}
+	tweakableRM := meta.NewDefaultRESTMapper(nil)
```

Comment:
trying to work out where this came from.

Reply:
yup. this is now reverted to original.